lib/takeaway.rb

```diff
@@ -0,0 +1,47 @@
+require "./lib/menu"
+
+class Takeaway
+
+attr_reader :order
+
+	def initialize
+	@menu = Menu.new
+	@dishes = @menu.dishes
+	@order = []
+	@total = 0
+	end
+
+	def list_menu
+	@dishes
+	end
+
+	def select_item(item, amount=1)
+		amount.times {order << item} 
+		"you added #{amount} #{item} to your basket"
+	end
+
+	def total
+	sum_item_total
+	"Your total is £#{@total}"
+	end
+
+
+private
+
+def sum_item_total
+@order.each do |item|
+	 		@dishes.select do |k,v| 
+	 		@total += v	if k == item
+			
+			end
+		end
```

Comment:
Lint/BlockAlignment: `end` at 37, 2 is not aligned with `@order.each do |item|` at 32, 0.